### PR TITLE
Help drawers

### PR DESCRIPTION
**Pull request by mplorentz**
#709 This adds help drawers to the 5 main tabs. The drawers are written in SwiftUI and have a custom navigation scheme for navigating between them. Some of them contain video and some contain links to the various tabs. 

[Figma](https://www.figma.com/file/6oTsbY3vHME19RNtgSGdAw/Planetary-Design-Plan?node-id=1479%3A24456)

### CODE REVIEW

File: Source/Controller/HelpDrawerCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+import UIKit
+import SwiftUI
+import Analytics
+import Logger
+
+/// A model for help UI that is presented in a sheet or popover.
+enum HelpDrawer {
+    case home, discover, notifications, hashtags, network
+    
+    init?(viewController: HelpDrawerHost) {
```
Comment (martindsq) [inline Source/Controller/HelpDrawerCoordinator.swift:18]: The chain of ifs makes me wonder if this enum should be better set by HelpDrawerHost instances. Like a property for instance.

**Review by martindsq — APPROVED** [review]
Awesome work! I left a minor comment though. I wonder if the gossip keyword in the last help drawer should point the user to somewhere where we explain what gossiping is?

File: Source/Controller/HelpDrawerCoordinator.swift (hunk 1 above)
Comment (mplorentz) [reply Source/Controller/HelpDrawerCoordinator.swift:18]: Ah yeah that's a good idea.

Comment (mplorentz) [reply Source/Controller/HelpDrawerCoordinator.swift]: Done.

**Comment by mplorentz** [comment]
@martindsq  
> I wonder if the gossip keyword in the last help drawer should point the user to somewhere where we explain what gossiping is?

I think that's a good idea, although I don't think such a page exists yet, and I don't want to take the time to make one right now. I opened #745 to do this in the future.

**Pull request merged**